advisories/github-reviewed/2024/03/GHSA-cxjh-pqwp-8mfp/GHSA-cxjh-pqwp-8mfp.json

```diff
@@ -66,10 +66,6 @@
     {
       "type": "PACKAGE",
       "url": "https://github.com/follow-redirects/follow-redirects"
-    },
-    {
-      "type": "WEB",
-      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/VOIF4EPQUCKDBEVTGRQDZ3CGTYQHPO7Z"
```

Comment:
Incorrect

Reply:
I think this is a UI bug on GH's end, I didn't touch the references section at all

Reply:
@RDIL We can make changes to the description without changing the reference links. Don't worry! 🙂